envi/archs/rx/disasm.py

```diff
@@ -0,0 +1,1349 @@
+import sys
+import copy
+import struct
+import logging
+
+import envi
+import envi.exc as e_exc
+import envi.bits as e_bits
+import envi.memory as e_mem
+
+from . import regs, rxtables
+from .const import *
```

Comment:
I still really don't like `import *`. It makes it hard for me to track where something is coming from and makes reviewing harder and I'd like to avoid its usage.